contrib/release/uploadrev

```diff
@@ -27,15 +27,6 @@ function perror() {
 }
 
 function configure_env() {
-  if ! git rev-parse $REV >/dev/null 2>&1; then
```

Comment:
This was removed because we don't rely on the Git branch for the release at all anymore for getting the binaries.